lib/systems/parse.nix

```diff
@@ -69,8 +69,13 @@ rec {
   cpuTypes = with significantBytes; setTypes types.openCpuType {
     arm      = { bits = 32; significantByte = littleEndian; family = "arm"; };
     armv5tel = { bits = 32; significantByte = littleEndian; family = "arm"; };
+    armv6a   = { bits = 32; significantByte = littleEndian; family = "arm"; };
```

Comment:
How are the R- and M- parts tested given that they probably don't run Linux at all or without a nommu-configured libc etc.?

Reply:
I merely wanted to use gccCrossStageStatic. This does allow gcc and binutils to configure.

Reply:
I also built binutils.